README.md

```diff
@@ -28,23 +39,44 @@ creates a master makefile for the project, and configures, builds, and installs
 all components. The result is an environment where you can work on individual
 components, then rebuild the project and test the installed artifacts.
 
-Runtime Requirements
----
-Vanagon is self-contained. A recent version of ruby (2.1 or greater) should be
-all that is required. Beyond that, ssh, rsync and git are also required on the
-host, and ssh-server and rsync is required on the target (package installation
-for the target can be customized in the platform config for the target).
+## Runtime Requirements
+
+Vanagon carries two sets of requirements: requirements for the local host where Vanagon is run, and for the remote target where compilation and packaging happens.
+
+Also, Vanagon ships with a number of engines which may include additional optional dependencies if you wish to use them. These engines are currently considered experimental, and receive less attention than the `Hardware` or `vmpooler` engines do. If you find a bug in these engines, [please open a ticket](https://tickets.puppetlabs.com/browse/PA) and let us know.
```

Comment:
welp, apparently that project is private. Should it be? Should we use CPR instead of PA for easier ticket routing?